sigstore-java/src/main/java/dev/sigstore/proto/ProtoMutators.java

```diff
@@ -59,4 +68,41 @@ public static HashAlgorithm from(Bundle.HashAlgorithm algorithm) {
     }
     throw new IllegalStateException("Unknown hash algorithm: " + algorithm);
   }
+
+  public static RekorEntry fromTLogEntry(TransparencyLogEntry tle) throws RekorParseException {
```

Comment:
I think I did a bad job with naming in this class. I think more appropriate name would be `toRekorEntry(...)`, I'll go update the other names after this PR.